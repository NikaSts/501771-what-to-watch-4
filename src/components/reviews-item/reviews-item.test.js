import React from 'react';
import renderer from 'react-test-renderer';
import ReviewsItem from './reviews-item';

const review = {
  id: 2,
  user: {
    id: 15,
    name: `Kendall`
  },
  rating: 8.7,
  comment: ``,
  date: ``
};

it(`ReviewsItem should render correctly`, () => {
  const tree = renderer
    .create(
        <ReviewsItem
          review={review}
        />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
