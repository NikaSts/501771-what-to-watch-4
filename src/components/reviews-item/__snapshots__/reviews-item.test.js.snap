// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsItem should render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        Kendall
      </cite>
      <time
        className="review__date"
        dateTime="Invalid date"
      >
        Invalid date
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8.7
  </div>
</div>
`;
